.github/workflows/cli_test.yaml

```diff
@@ -58,7 +58,8 @@ jobs:
 
       - name: Install ginkgo
         run: |
-          go install github.com/onsi/ginkgo/v2/ginkgo@latest
+          go get github.com/onsi/ginkgo/v2@v2.11.0
```

Comment:
Don't need this line

Reply:
I added it because it is suggested in the docs. Should we remove it?

https://onsi.github.io/ginkgo/#installing-ginkgo

Reply:
Yes, we should remove it. We just need ginkgo binary